mfr/extensions/tabular/libs/stdlib_tools.py

```diff
@@ -6,12 +7,15 @@ def csv_stdlib(fp):
     :param fp: File pointer object
     :return: tuple of table headers and data
     """
+    data = fp.read(2048)
+    fp.seek(0)
+
     try:
-        dialect = csv.Sniffer().sniff(fp.read(2048))
+        dialect = csv.Sniffer().sniff(data)
     except csv.Error:
         dialect = csv.excel
 
-    fp.seek(0)
+    dialect = detect_quoted_fields(dialect, data)
```

Comment:
Let's move this line to an `else` block on the `try...except`, and remove the conditional in `detect_quoted_fields`.